frontend/src/AlertsManagement/AlertsList.js

```diff
@@ -47,7 +47,7 @@ export default function AlertsList(props) {
                 disableTypography
                 primary={<Typography variant="body1" >Alert on 
                   <Box fontWeight='fontWeightBold' display='inline' m={1} style={{ color: '#0FA3B1'}}>
-                    {` ${allAlerts.get(alertId).timestampDate} (${formatDate(allAlerts.get(alertId).timestampDate)})`}
+                    {` ${allAlerts.get(alertId).timestampDate} (${formatDate(allAlerts.get(alertId).timestampDate, true)})`}
```

Comment:
nit: i don't think its good passing in `boolean` to a function, since it is less readable and arbitrary what the `boolean` stands for. (paula mentioned this before in one of my PRs, not sure if completely applicable to js code thou)

or if a `boolean` is necessary, maybe you can do:
 `formatDate(allAlerts.get(alertId).timestampDate, /** mFirst = */ true)`

Reply:
that's a good point. I'll add the comment